Dockerfile

```
@@ -0,0 +1,18 @@
+FROM golang:1.12 as builder
+
+WORKDIR /go/src/github.com/mattdamon108/go-graphql-api-boilerplate
```

I prefer to use `go modules`. How about changing to enable to use `go modules`?

```suggestion
WORKDIR /usr/app
```